integration-tests/gradle/src/test/resources/jandex-basic-multi-module-project/settings.gradle

```diff
@@ -0,0 +1,18 @@
+pluginManagement {
+    repositories {
+        jcenter()
+        mavenLocal()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+    //noinspection GroovyAssignabilityCheck
```

Comment:
I assume this comment can go away :)

Reply:
This was copied as it is from basic-multi-module-project template, but I'll remove it